dsn.go

```diff
@@ -42,15 +42,16 @@ type Config struct {
 	ReadTimeout  time.Duration     // I/O read timeout
 	WriteTimeout time.Duration     // I/O write timeout
 
-	AllowAllFiles           bool // Allow all files to be used with LOAD DATA LOCAL INFILE
-	AllowCleartextPasswords bool // Allows the cleartext client side plugin
-	AllowOldPasswords       bool // Allows the old insecure password method
-	ClientFoundRows         bool // Return number of matching rows instead of rows changed
-	ColumnsWithAlias        bool // Prepend table alias to column names
-	InterpolateParams       bool // Interpolate placeholders into query string
-	MultiStatements         bool // Allow multiple statements in one query
-	ParseTime               bool // Parse time values to time.Time
-	Strict                  bool // Return warnings as errors
+	AllowAllFiles           bool              // Allow all files to be used with LOAD DATA LOCAL INFILE
+	AllowCleartextPasswords bool              // Allows the cleartext client side plugin
+	AllowOldPasswords       bool              // Allows the old insecure password method
+	ClientFoundRows         bool              // Return number of matching rows instead of rows changed
+	ColumnsWithAlias        bool              // Prepend table alias to column names
+	InterpolateParams       bool              // Interpolate placeholders into query string
+	MultiStatements         bool              // Allow multiple statements in one query
+	ParseTime               bool              // Parse time values to time.Time
+	Strict                  bool              // Return warnings as errors
+	ConnAttrs               map[string]string // Connection Attributes
```

Comment:
forget what I said, this should stay user-friendly. The Config struct is now exported